### PR TITLE
Add Team interface support documentation

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -207,6 +207,22 @@ interfaces:
           stp-priority: 32
 ```
 
+## Interfaces: Team
+
+```yaml
+---
+interfaces:
+- name: team
```
Comment (tyll) [inline examples.md]: Please call this `team0`

Comment (ffmancera) [reply examples.md]: Ops, done. Thanks!

**Comment by ffmancera** [comment]
@tyll could we merge this? :-)

File: devel/api.md [hunk 2]
```diff
@@ -881,6 +889,56 @@ Type: `integer`
...
+            Team.RUNNER_SUBTREE: {
+                Team.Runner.NAME: Team.Runner.RunnerMode.LOAD_BALANCE
+            }
+        }
+    ]
+}
+```
+
+These values are based on the teamd JSON API. For further information please
+refer to [teamd manual page](https://www.mankier.com/5/teamd.conf)
```
Comment (cathay4t) [inline devel/api.md:925]: I would not suggest to use external third party website for manpage, you may just say ` man 5 teamd.conf` or the upstream link
of that manpage: https://github.com/jpirko/libteam/blob/master/man/teamd.conf.5

**Review by cathay4t — CHANGES_REQUESTED** [review]

**Pull request closed without merging**